uwblueprint/supportive-housing
# Login Page Skeleton

## Notion task link


[Login Page Skeleton](https://www.notion.so/uwblueprintexecs/Login-Page-Skeleton-d58ed370f9374ee088ea688264b59bf0?pvs=4)



## Implementation description
* Rewrote all components in Chakra UI for easier design + app wide consistency
* Matched the Figma designs found [here](https://www.figma.com/file/woRXyIloJYllaqE62wjgYv/Login%2F2FA?type=design&node-id=608-2&mode=design&t=eD8rGdu8rrzRXzSy-0)




## Steps to test
1. Launch app, if you are brought to homepage click 'Logout'. 
2. You should be able to login with a correct user / pass combination
3. You should not be able to login with an incorrect user / pass combination
4. You should be able to visit the Sign Up page



## What should reviewers focus on?
* I would like to know how this design scales on different size monitors / displays. Currently all containers are dynamically sized with user's viewport. However, all text sizes have been hardcoded with pixel values.


## Checklist
